cmd/joy/release.go

```diff
@@ -281,6 +285,69 @@ func NewReleaseRenderCmd() *cobra.Command {
 	return cmd
 }
 
+func NewValidateCommand() *cobra.Command {
+	var env string
+
+	cmd := &cobra.Command{
+		Use:   "validate [releases...]",
+		Short: "validate releases",
+		Args:  cobra.RangeArgs(0, 1),
+		RunE: func(cmd *cobra.Command, args []string) error {
+			cfg := config.FromContext(cmd.Context())
+
+			selectedEnvs := func() []string {
+				if env == "" {
+					return cfg.Environments.Selected
+				}
+				return strings.Split(env, ",")
+			}()
+
+			releaseFilter := func() filtering.Filter {
+				if len(args) == 0 {
+					return nil
+				}
+				return filtering.NewSpecificReleasesFilter(args)
+			}()
+
+			// Load catalog
```

Comment:
My favorite comment of all! I'm trying to remember the exact words a friend once told me...